web/static/js/components/surveys/SurveyEdit.jsx

```diff
@@ -34,19 +31,11 @@ class SurveyEdit extends Component {
   componentWillMount() {
```

Comment:
I think we should create a tech-debt-issue to move "fetching elements" from `componentWillMount` to `componentDidMount` (or vice versa) so that we are consistent across all components.

Reply:
We can add this to #1966 since `componentWillMount()` is deprecated anyway.